std/http/mod.ts

```diff
@@ -1,3 +1,4 @@
 export * from "./cookie.ts";
 export * from "./http_status.ts";
+export * from "./http_method.ts";
```

Comment:
I am not sure if exporting files should follow alphabetical order, but if it did then `./http_method.ts` should be exported first.